README.md

```diff
@@ -32,20 +33,27 @@ To get a local copy up and running:
 
 👤 **Ademola Adebayo**
 
-- Github: [@Ademola]()
+- Github: [@Ademola](https://github.com/Smart1-hub)
 
-- Twitter: [@ademola_adebayo]()
+- Twitter: [@ademola_adebayo](https://twitter.com/ademola_adebayo)
 
-- LinkedIn: [Ademola Adebayo](https://www.linkedin.com/in/)
+- LinkedIn: [Ademola Adebayo](https://www.linkedin.com/in/ademola-adebayo-81051578/)
 
 - Loom: 
+https://www.loom.com/share/4bae1adf0d6e49c6afcd492efdf32c24
 
 ## Design By
 
 👤 **Cindy Shin**
 
 - Behance: https://www.behance.net/gallery/29845175/CC-Global-Summit-2015
 
+## Acknowledgement
+
+Microvers Org.
+Cindy Shin
+All whose codes have been built on
+
 ## Contributing
 
 Contributions, issues, and feature requests are welcome!
```

Comment:
- [ ] Please fix the MIT link in the README file for a completely professional README file